wurst/_handles/Quest.wurst

```diff
@@ -0,0 +1,86 @@
+package Quest
+
+public function quest.setRequired(boolean req)
+    QuestSetRequired(this, req)
+
+public function quest.setTitle(string title)
+    QuestSetTitle(this, title)
+
+public function quest.setDescription(string text)
+    QuestSetDescription(this, text)
+
+public function quest.setIcon(string path)
+    QuestSetIconPath(this, path)
+
+public function quest.completed(boolean b)
+    QuestSetCompleted(this, b)
+
+public function quest.failed(boolean b)
+    QuestSetFailed(this, b)
+
+public function quest.discovered(boolean b)
+    QuestSetDiscovered(this, b)
+
+public function quest.isEnabled() returns boolean
+    return IsQuestEnabled(this)
+
+public function quest.isCompleted() returns boolean
+    return IsQuestCompleted(this)
+
+public function quest.flash()
+    FlashQuestDialogButton()
+
+public function quest.remove()
+    DestroyQuest(this)
+
+
+    public enum QuestState
```

Comment:
"public enum QuestState" should not be indented, this is causing the travis build to fail